app/models/api/members_api/client.rb

```diff
@@ -24,7 +24,14 @@ def constituency(search_term)
   end
 
   def member_for_constituency(constituency)
-    member_id = member_id(constituency).object
+    member = member_id(constituency)
+
+    if member.error.present?
+      Sentry.capture_message(member.error.message, level: :warning)
```

Comment:
We should probably try and use Application Insights instead of Sentry here: https://github.com/Microsoft/ApplicationInsights-Ruby#sending-a-simple-event-telemetry-item